waterlog/src/Components/AlertTable.js

```diff
@@ -26,31 +29,36 @@ class AlertTableComponent extends Component {
         <table>
           <tbody>
             {alerts.map((alert, index) => (
+              <Card className="card-alert">
+              <CardContent>
               <tr
                 key={index}
-                className="table-row table-row-unseen"
-                onClick={() =>
-                  (window.location.href = `alert/${alert.entityName}/${
+                className={alert.status === 1 ? "entity-disabled table-row" : "table-row table-row-unseen"}
+                onClick={alert.status === 2 ? () =>
+                (window.location.href = `alert/${alert.entityName}/${
                     alert.entityId
                   }`)
+                : () => {}
                 }
               >
                 <td className="leak-date">{formatDate(alert.date)}</td>
-                <td>{`${alert.entityName.toUpperCase()} ${
+                <td className="leak-name">{`${alert.entityName.toUpperCase()} ${
                   alert.entityId
                 } ${alert.entityType.toUpperCase()}`}</td>
-                <td>
-                  {alert.entityName === "Segment" ? `R${alert.cost}/hr` : ""}
+                <td className="leak-cost">
+                  {alert.entityName === "Segment" ? `R${(alert.cost).toFixed(2)}/hr` : ""}
                 </td>
-                <td>
+                <td className="leak-severity">
                   {alert.entityName === "Segment"
                     ? getStatusIcon(alert.severity)
                     : ""}
                 </td>
               </tr>
+              </CardContent>
+            </Card>
             ))}
-          </tbody>
-        </table>
+            </tbody>
+          </table>
```

Comment:
The usage of table is not advised with the power of css3 and html5.
If this is a financial system, it is fine but you can use div and other html5 elements to style without using table